src/Okta.Sdk/DefaultRetryStrategy.cs

```diff
@@ -146,5 +151,42 @@ private TimeSpan CalculateDelay(HttpResponseMessage response)
 
             return backoffSeconds;
         }
+
+        private static async Task<HttpRequestMessage> CloneHttpRequestMessageAsync(HttpRequestMessage request)
+        {
+            HttpRequestMessage clonedRequest = new HttpRequestMessage(request.Method, request.RequestUri);
+
+            if (request.Content != null)
+            {
+                // Copy the request's content (via a MemoryStream) into the cloned object
+                var memoryStream = new MemoryStream();
+                await request.Content.CopyToAsync(memoryStream).ConfigureAwait(false);
```

Comment:
It's not possible, because if you try to clone a request with content which was used already, it will fail with a `Cannot access a disposed object.` error.

Reply:
Good to know, but also a bit strange. Because if the `IHttpContent` is already disposed of, then shouldn't we also be getting an error when we call `CopyToAsync` here?

Sometimes the .NET framework does weird things and you just have to accept it. I guess this is one of those times.

Up to you but it might be worth explaining this in a comment for future readers.